File: C/Assignment-2/Prog14.c
#include<stdio.h>

int main(){
    int X = 0, Y = 0;
    printf("Enter a number: ");
    scanf("%d", &X);
    
    switch (X)
    {
    case 6:
        Y = X + 10;
        break;
    case 7:
        Y = X*X;
        break;
    case 12:
        Y = 2*X + 4;
    default:
        Y = X*6 - 1;
        break;
    }
    return 0;
}

File: Python/Assignment-2/Prog14.py
def main():
    X = int(input("Enter a number: "))
    Y = 0

    match X:
        case 6:
            Y = X + 10
        case 7:
            Y = X * X
        case 12:
            Y = 2 * X + 4
        case _:
            Y = X * 6 - 1

    print("Y =", Y)

if __name__ == "__main__":
    main()
